### PR TITLE
chore: market path

### LINKED ISSUE #264: Add market path
add path to market icon

### PR DESCRIPTION
**Pull request by ksuhara**
close #264

### CODE REVIEW

File: packages/shared/src/configs/markets.json [hunk 1]
```diff
@@ -5,6 +5,24 @@
     "url": {
       "mainnet": "https://opensea.io/assets/",
       "testnet": "https://testnets.opensea.io/assets/"
+    },
+    "path": {
```
Comment (taijusanagi) [inline packages/shared/src/configs/markets.json:9]: This is nice, but a bit seems that does not DRY with network configs.

File: packages/shared/src/configs/networks.json [hunk 2]
```diff
@@ -85,7 +85,7 @@
     "rpc": "https://rpc.gnosischain.com/",
     "explorer": "https://blockscout.com/xdai/mainnet/",
     "protocols": ["seaport"],
-    "markets": ["tofu"],
```
Comment (taijusanagi) [inline packages/shared/src/configs/networks.json:88]: thanks!

File: packages/next-app/src/components/organisms/Asset/useMarketIcons.tsx [hunk 3]
```diff
@@ -13,11 +13,11 @@ export const useMarketIcons = (chainId: string) => {
     return networks[chainId].markets
       .map((market: string) => {
         if (!isMarket(market)) {
-          return "";
+          return { icon: "", url: "" };
         }
-        return markets[market].icon;
+        return { icon: markets[market].icon, url: `${markets[market].url.mainnet}/${markets[market].path[chainId]}` };
```
Comment (taijusanagi) [inline packages/next-app/src/components/organisms/Asset/useMarketIcons.tsx:18]: I prefer this way that putting logics in hooks, good!

File: packages/next-app/src/components/organisms/Asset/Asset.tsx [hunk 4]
```diff
@@ -92,7 +92,10 @@ export const Asset: React.FC<AssetProps> = ({ asset }) => {
         src={`/icons/networks/${networks[asset.chainId as ChainId].icon}`}
         alt="Dan Abramov"
       />
-      <Link href={`${networks[asset.chainId as ChainId].explorer}/address/${asset.contractAddress}`}>
+      <Link
+        href={`${networks[asset.chainId as ChainId].explorer}/address/${asset.contractAddress}`}
+        chakraLinkProps={{ isExternal: true }}
```
Comment (taijusanagi) [inline packages/next-app/src/components/organisms/Asset/Asset.tsx:97]: thanks!

File: packages/shared/src/configs/markets.json (hunk 1 above)
Comment (ksuhara) [reply packages/shared/src/configs/markets.json:9]: agree but couldn't come up with a better way. let me merge it for now

**Pull request merged**